protocol/messenger_communities.go

```diff
@@ -576,6 +576,16 @@ func (m *Messenger) JoinedCommunities() ([]*communities.Community, error) {
 	return m.communitiesManager.Joined()
 }
 
+func (m *Messenger) CommunityUpdateLastOpenedAt(communityID string) (*MessengerResponse, error) {
+	updatedCommunity, err := m.communitiesManager.CommunityUpdateLastOpenedAt(communityID)
+	if err != nil {
+		return nil, err
+	}
+	response := &MessengerResponse{}
+	response.AddCommunity(updatedCommunity)
+	return response, nil
```

Comment:
I believe we should call smth like `SyncCommunity` here. Otherwise no sync message is sent.

Reply:
Done